src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java

```diff
@@ -253,13 +254,21 @@ protected Details getTransactionDetails(@Nonnull CtpPaymentWithCart paymentWithC
      * @see #mapCustomLineItemToPaypalPlusItem(CustomLineItem, List)
      */
     protected Stream<Item> mapCustomLineItemToPaypalPlusItem(@Nonnull CustomLineItem customLineItem, @Nonnull List<Locale> locales) {
+        String itemName = customLineItem.getName().get(locales);
+        if (StringUtils.isBlank(itemName)) {
+            Locale fallBackLocale = customLineItem.getName().getLocales()
+                    .iterator().next();
```

Comment:
Any chance we could run into NPE here?

Reply:
It is possible that we could have ElementNotFound exception when the `iterator()` returns empty `Set`. I have handled it the modification here. 00b063221be5c47c9909e907517a4c462ddf4167

Reply:
is won't be null because of localized get?

Reply:
> is won't be null because of localized get?

`StringUtils.isBlank(itemName)` will handle it.